walk-in-interview/src/main/webapp/account/firebase-auth.js

```diff
@@ -67,20 +67,81 @@ Auth.signIntoBusinessAccount = (email, password) => {
               // Session login endpoint is queried and session cookie is set.
               // CSRF protection should be taken into account.
               const csrfToken = getCookie('csrfToken');
-              return Auth.postIdTokenToSessionLogin(API['business-log-in'],
+              return Auth.postIdTokenToSessionLogin(API['log-in'],
                   idToken, csrfToken);
             });
       });
 };
 
+/**
+ * This will create a new applicant account.
+ *
+ * @param {String} elementId The div element to add the UI.
+ * @param {String} successPath The url for redirect on login success.
+ * @param {String} newUserInfo The message to be displayed if the it is a new user.
+ */
+Auth.createApplicantAccount = (elementId, successPath, newUserInfo) => {
+  Auth.addPhoneAuthUI(elementId, successPath, newUserInfo);
+
+  // Send id token to the backend
+  var user = firebase.auth().currentUser;
+  if (user) {
+    // User is signed in.
+    
+    // Get the user's ID token as it is needed to exchange
+    // for a session cookie.
+    return user.getIdToken()
+    .then((idToken) => {
+      // Session login endpoint is queried and session cookie is set.
+      // CSRF protection should be taken into account.
+      const csrfToken = getCookie('csrfToken');
```

Comment:
don't you need to check the return value here? (and every other method reading cookie)

Reply:
`getCookie` will return empty string if no cookie found, so it is ok in syntax